src/opnsense/service/templates/OPNsense/Unbound/core/+TARGETS

```diff
@@ -5,3 +5,4 @@ private_domains.conf:/var/unbound/private_domains.conf
 domainoverrides.conf:/usr/local/etc/unbound.opnsense.d/domainoverrides.conf
 root.min.hints:/var/unbound/root.hints
 unbound_dhcpd.conf:/usr/local/etc/unbound_dhcpd.conf
+dnsbl_module.py:/var/unbound/dnsbl_module.py
```

Comment:
One could debate if code should be stashed in templates, but I'm not sure about any better location either at the moment. We can always move it later on if needed.

Reply:
Agreed, it does not happen often (if at all). If the use case would turn out bigger we could always define a separate mechanism.